Automattic/jetpack
# VideoPress Block: Fix video display at any aspect ratio to not show black bars

Fixes https://github.com/Automattic/wp-calypso/issues/54351

There's been a number of fixes put in the VideoPress block to fix issues with certain aspect ratios causing black bars at the top/bottom/sides of the video. The block uses the VideoPress iframe embed to display the video, and the iframe embed was not configured to resize within its parent container. Instead it expects a width and height to be provided to it in the same way that you would embed a YouTube video with a fixed height and width. 

To get around this we've added support for a new url argument to be passed to the VideoPress oembed url that will have the `videopress-iframe.js` always resize the video to its parent container. This way we get nice looking embeds in gutenberg as well as on the front end:

<img width="832" alt="Screen Shot 2021-08-06 at 4 05 35 PM" src="https://user-images.githubusercontent.com/789137/128575440-2607b473-6c25-4e07-8da8-526febb7bd01.png">

#### Changes proposed in this Pull Request:
* The `has-aspect-ratio` classes will no longer be added since they are no longer needed (I've left the code for this just in case we need to roll back to it)
* `resizeToParent` VideoPress url argument will always be added to get the resizing working nicely.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Insert some videos into a post (both from media library and a new upload directly to editor), they should not have any large black bars on any side (Note: Some browsers have a 1px bar sometimes - we are looking into this).
* Try inserting videos into a `Columns` block, they also should not have large black bars at the sides.
* Publish the post and the videos should look the same as in the editor.
